core/server/master/pom.xml

```diff
@@ -85,6 +85,12 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
We leverage the local under file system in some integration tests. Another approach would be to include local under filesystem in common as the implementation of the under file system API.

Previously we didn't have this problem because the client brought in ufs dependencies.